src/Integration.Vsix/Commands/HelpMenu/HelpMenu.vsct

```diff
@@ -5,34 +5,40 @@
 
   <Commands package="guidVsPackagePkgString">
     <Groups>
-      <Group guid="HelpMenuCmdSet" id="HelpGroup">
+      <Group guid="HelpMenuCmdSet" id="HelpGroupMain">
```

Comment:
I think our convention is `blahGroup`, so it would be better to rename these to HelpMainGroup, HelpLogsGroup, HelpAboutGroup